extensions/mssql/src/main.ts

```diff
@@ -187,6 +199,30 @@ export async function activate(context: vscode.ExtensionContext): Promise<MssqlE
 		return null;
 	}
 
+	function getFriendlyEndpointNames(name: string): string {
+		let friendlyName: string = name;
+		switch (name) {
+			case 'app-proxy':
+				friendlyName = 'Application Proxy';
```

Comment:
Anything we're giving names for should be localized. Let's use the localize function here.

Reply:
Added the localization